lib/loggers/ProjectBuildLogger.js

```
@@ -0,0 +1,62 @@
+import AbstractLogger from "./AbstractLogger.js";
+
+class ProjectBuildLogger extends AbstractLogger {
+	#projectName;
+	#projectType;
+	#tasksToRun;
+
+	constructor({projectName, projectType}) {
+		super();
+		this.#projectName = projectName;
+		this.#projectType = projectType;
+	}
+
+	setTasks(tasksToRun) {
+		this.#tasksToRun = tasksToRun;
+
+		this._emit("ui5-project-build-status", {
+			projectName: this.#projectName,
+			projectType: this.#projectType,
+			tasksToRun
+		});
+	}
+
+	startTask(taskName) {
+		if (!this.#tasksToRun || !this.#tasksToRun.includes(taskName)) {
+			throw new Error(`ProjectBuildLogger.startTask: Unknown task ${taskName}`);
```

Do we really need to throw an error for those cases?
I was wondering, because if there's a task/build/configuration/etc. with a wrong name, then it should fail either way. 
Of course, we need to inform the user/developer that there could be something wrong, but I'm missing why we need to fail the whole thing if, for example, there's a typo for the logger's argument. To me, this seems somehow like coupling with external modules